packages/client/src/raffles/types/getRaffleParticipation.types.ts

```diff
@@ -0,0 +1,8 @@
+import type { Config } from '../..';
+import type { Raffle, RaffleParticipation } from './raffles.types';
+
+export type GetRaffleParticipation = (
+  raffleId: Raffle['id'],
+  participationId: string,
+  config?: Config,
+) => Promise<RaffleParticipation[]>;
```

Comment:
Is the response an array? According to swagger it's a single `RaffleParticipation`

Reply:
Right, but I spoke with @rc23 and he said that a raffle can have multiple participations...

Reply:
But you're fetching a single participation (with a `participationId`) here... the response doesn't really make sense and it's also not on swagger. Have you tried to validate this using postman for example? You can also double check with the backend team.